dashboard/src/pages/localhost/LocalhostAttendanceProcess.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <Dialog
+    class="z-50"
+    v-model="showDialog"
+    :options="{
+      title: 'Error',
+      message: dialogMessage,
+    }"
+  />
+  <Header />
+  <div class="w-full h-screen flex justify-center">
+    <div v-if="transferStatus" class="max-w-screen-xl w-full flex flex-col items-start">
+        <div class="flex flex-col justify-between md:flex-row md:items-center w-full p-2">
+            <div>
+                <HackathonHeader v-if="hackathon_doc.data" :hackathon="hackathon_doc" :showBanner="false"/>
+            </div>
+            <div class="flex flex-col md:items-end">
+                <LocalhostHeader v-if="localhost_doc.data" :localhost="localhost_doc.data"/>
+            </div>
+        </div>
+        <StatusMessage :status="transferStatus" />
+    </div>
+    <div v-else class="flex w-full justify-center items-center">
+        <LoadingIndicator class="w-6"/>
+    </div>
+  </div>
+</template>
+<script setup>
+import Header from '@/components/Header.vue'
+import StatusMessage from '@/components/localhost/StatusMessage.vue'
+import HackathonHeader from '@/components/hackathon/HackathonParticipantHeader.vue'
+import LocalhostHeader from '@/components/localhost/LocalhostHeader.vue'
+import { createDocumentResource, createResource, Dialog, usePageMeta, LoadingIndicator } from 'frappe-ui'
+import { onMounted, ref } from 'vue'
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+
+const participant = route.query.participant
+const localhost = route.query.localhost
```

Comment:
This query parameter exposes the localhost ID to the participant. 

With the All permissions set for LocalHost doctype to `read`, and `select`. - the organiser dashboard is visible for the participant if followed the URL pattern -> /dashboard/localhost/id. 

Adding a validation / revoking All permissions would be needed to limit access for anyone to this page.

Reply:
The participant is unable to make changes - like `accept/reject` other participants - due to lack of write permissions